rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -0,0 +1,110 @@
+from dataclasses import dataclass
+from typing import List, Optional
+from typing import Dict, Generic, List, Optional, Tuple, Type, TypeVar
+
+import stp.action as action
+import stp.rc as rc
+import stp.tactic as tactic
+import stp.role as role
+
+import rj_gameplay.eval
+import rj_gameplay.skill as skills
+from rj_gameplay.skill import pivot_kick, receive
+import stp.skill as skill
+import numpy as np
+
+
+class pass_cost(role.CostFn):
```

Comment:
IMO naming this "passee_cost" or something that mirrors the "passer_cost" function below would be clearer. Up to you